packages/manager/styles/globals.css

```diff
@@ -71,3 +72,29 @@ button.spaship_btn {
 * {
   box-sizing: border-box;
 }
+
+.spaship-circle {
+  border-radius: 999px;
+  width: 70px;
+  height: 70px;
+  margin-top: 25px;
+  margin-left: 25px;
+  background: hsl(42, 98%, 54%);
```

Comment:
@soumyadip007 can we create variables instead, would be easier to manage in the future